iwyu_output.cc

```diff
@@ -730,6 +716,10 @@ static void LogIncludeMapping(const string& reason, const OneUse& use) {
 namespace internal {
 
 bool DeclCanBeForwardDeclared(const Decl* decl) {
```

Comment:
The logging on line 1016 is now a little misleading -- it basically says "no forward-declare, because not a class". Now there are more nuanced reasons for why a forward-declare might be disabled (and I think we should consider allowing forward-declaring free functions here at some point).